edx_solutions_api_integration/users/tests.py

```diff
@@ -1742,6 +1743,63 @@ def test_user_organizations_list(self):
         response = self.do_get('{}/4356340/organizations/'.format(self.users_base_uri))
         self.assertEqual(response.status_code, 404)
 
+    def test_mobileapps_in_organizations_list(self):
```

Comment:
These unit test should be written in `mobile_api/tests` since we are adding themes and mobileapps support in `api/server/mobile/v1/users/organizations/?username=<username>` API.